strawberry/flask/views.py

```diff
@@ -92,3 +92,55 @@ def dispatch_request(self) -> Response:
         response.set_data(json.dumps(response_data))
 
         return response
+
+
+class AsyncGraphQLView(GraphQLView):
+    async def dispatch_request(self):
+        method = request.method
+        content_type = request.content_type or ""
+
+        if "application/json" in content_type:
+            data: dict = request.json  # type:ignore[assignment]
+        elif content_type.startswith("multipart/form-data"):
+            operations = json.loads(request.form.get("operations", "{}"))
+            files_map = json.loads(request.form.get("map", "{}"))
+
```

Comment:
missing coverage here is fine, it should be improved by my PR: https://github.com/strawberry-graphql/strawberry/pull/1840 
so no worries about this